client/my-sites/activity/activity-log-tasklist/to-update.jsx

```diff
@@ -15,6 +15,11 @@ import { requestSiteAlerts } from 'calypso/state/data-getters';
 
 const emptyList = [];
 
+const unionBySlug = ( a = [], b = [] ) => [
+	...a,
+	...b.filter( ( { slug: bSlug } ) => ! a.some( ( { slug: aSlug } ) => aSlug === bSlug ) ),
```

Comment:
style nit: destructuring makes things a bit worse here IMO.
```js
bs.filter( ( b ) => ! as.some( ( a ) => a.slug === b.slug ) )
```
is more concise.

Reply:
I think you're right 🙈  Fixed in 65d759f5957b9acd5f9869a97ec3b44b8f5f8310